helpers.sh

```diff
@@ -74,21 +74,29 @@ function repo_sync {
 		status "Adding git config extraheader for $domain/factories"
 		git config --global http.https://${domain}/factories.extraheader "$(cat /secrets/git.http.extraheader)"
 	fi
+	_repo_extra_args = ""
 	for i in $(seq 4); do
-		run repo init --repo-rev=v2.35 --no-clone-bundle -u $* ${REPO_INIT_OVERRIDES} && break
+		run repo init $_repo_extra_args --repo-rev=v2.35 --no-clone-bundle -u $* ${REPO_INIT_OVERRIDES} && break
+		_repo_extra_args = "--verbose"
```

Comment:
What is the downside of always running with verbose?

Reply:
basically it just generates less noise, as the first time it almost always works